pkg/apis/kubernetes/v1/example_test.go

```diff
@@ -0,0 +1,37 @@
+package v1
+
+import (
+	"context"
+	"log"
+
+	"go.anx.io/go-anxcloud/pkg/api"
+	"go.anx.io/go-anxcloud/pkg/client"
+	"go.anx.io/go-anxcloud/pkg/utils/pointer"
+)
+
+func Example() {
```

Comment:
will `go test` run this example? This would fail when not having `ANEXIA_TOKEN` set, right?

Reply:
This example does not run with `go test`, as it does not have the "output" comment. Yes, AuthFromEnv returns an error if the token is not set.